src/pages/WorkProgress.tsx

```diff
@@ -161,10 +162,24 @@ const WorkProgress = ({ workTypes }: { workTypes: string[] }) => {
   return (
     <AppShell>
       <AppShell.Header>
-        <AppShell.Title>STAN</AppShell.Title>
+        <div className="flex flex-row">
```

Comment:
Assume this is something needed so the summary dashboard doesnt get too close to the header, just checking if its actually needed or not

Reply:
In fact, this is not required. I have removed it. Good finding 👍